tests/jobs/test_models.py

```diff
@@ -160,6 +161,21 @@ def test_sync_terminated(request, mocker, sparkjob_provisioner_mocks,
     assert spark_job.latest_run.finished_at == now
     assert not spark_job.latest_run.alerts.exists()
 
+    metric = Metric.objects.get(key='sparkjob-normalized-instance-hours')
```

Comment:
Let's check here that there is exactly one `Metric` with that name and not use `get` to fetch it.

Reply:
Curious why? The `get` would fail if there are not exactly 1.

Reply:
Right, it's just a nicer assertion than having to decrypt why it would be returning a `MultipleObjectsReturned` exception